action.yml

```diff
@@ -61,6 +74,7 @@ runs:
         EOF
 
     - id: render
+      if: steps.enable_guides.outputs.GUIDES == 'false'
```

Comment:
Can GUIDES be false? It is only set to true above. Shouldn't it a GUIDES != 'true'?

Reply:
see line 48, it is set to string 'true' or 'false' there

Reply:
Now it is false there. AFAIR it was true before.